blocks/library/quote/index.js

```diff
@@ -20,11 +20,21 @@ registerBlock( 'core/quote', {
 		citation: html( 'footer' )
 	},
 
+	controls: [ '1', '2' ].map( ( level ) => ( {
```

Comment:
Not sure `level` applies as well here as it does for a heading. `variant`, `variation`, or `style` might be more accurate.